plans/upgrade.pp

```diff
@@ -15,14 +15,14 @@
 plan peadm::upgrade (
   # Standard
   Peadm::SingleTargetSpec           $primary_host,
-  Optional[Peadm::SingleTargetSpec] $primary_replica_host = undef,
+  Optional[Peadm::SingleTargetSpec] $replica_host = undef,
 
   # Large
   Optional[TargetSpec]              $compiler_hosts      = undef,
 
   # Extra Large
-  Optional[Peadm::SingleTargetSpec] $puppetdb_database_host         = undef,
-  Optional[Peadm::SingleTargetSpec] $puppetdb_database_replica_host = undef,
+  Optional[Peadm::SingleTargetSpec] $primary_postgresql_host         = undef,
```

Comment:
Same here should the = line up for indentation

Reply:
Fixed.